apps/src/applab/designMode.js

```diff
@@ -216,7 +216,18 @@ designMode.onPropertyChange = function(element, name, value) {
       element.setAttribute('rows', value);
       break;
     default:
-      throw "unknown property name " + name;
+      // Mark as unhandled, but give typeSpecificPropertyChange a chance to
+      // handle it
+      handled = false;
+  }
+
+  if (elementLibrary.typeSpecificPropertyChange(element, name, value)) {
+    designMode.editElementProperties(element);
+    handled = true;
```

Comment:
seems like we should mark this as handled in the case where a typeSpecificPropertyChange handler is found, but no change is made.